src/t8_schemes/t8_default/t8_default_common_cxx.cxx

```diff
@@ -127,17 +127,36 @@ t8_gloidx_t
   t8_default_scheme_common_c::t8_element_count_leafs (const t8_element_t * t,
                                                       int level)
 {
-#if T8_ENABLE_DEBUG
-  if (eclass == T8_ECLASS_PYRAMID) {
-    SC_ABORT
-      ("Cannot compute leaf count of pyramids via the default_common implementation");
-  }
-#endif
+
+
+
   int                 element_level = t8_element_level (t);
   int                 dim = t8_eclass_to_dimension[eclass];
+  if (eclass == T8_ECLASS_PYRAMID) {
+      int level_diff = level - element_level;
+      return element_level > level ? 0 :
+                                     2 * sc_intpow64(8,level_diff) -
+                                     sc_intpow64(6, level_diff);
+  }
   return count_leafs_from_level (element_level, level, dim);
 }
 
+/* Count the maximum number of siblings.
+ * The number of children is 2^dim for each element, except for pyramids.
+ */
+/* *INDENT-OFF* */
+/* Indent bug: indent adds an additional const */
+int
+t8_default_scheme_common_c::t8_element_max_num_siblings (const t8_element_t * elem) const
+/* *INDENT-ON* */
+{
+  const int           dim = t8_eclass_to_dimension[eclass];
+  if (eclass == T8_ECLASS_PYRAMID) {
+    return 10;
```

Comment:
This does not match the documentation of the function.
"max num siblings of element or any descendant".
If the element is a tet of class pyra, then following the documentation the return value should be 8.

Either change the documentation or the behaviour of the function.

Reply:
I changed the documentation of the function